server/src/models/OpenStax/H5PEditor.spec.ts

```diff
@@ -77,6 +79,21 @@ describe('H5PEditor', () => {
       it(`does not mutate semantics and alters the correct values (${fakeLib.machineName})`, () => {
         const serialized = JSON.stringify(fakeSemantics);
         const altered = alterLibrarySemantics(fakeLib, fakeSemantics);
+        walkJSON(altered, (field) => {
```

Comment:
This is only here to try to make the snapshots a bit less verbose: the tag array is pretty large and was being duplicated for each entry.